Modules/SubModule_Population/xASL_wrp_CreatePopulationTemplates.m

```diff
@@ -334,8 +338,8 @@ function xASL_wrp_CreatePopulationTemplates(x, bSaveUnmasked, bCompute4Sets, Spe
             % ----------------------------------------------------------------------------------------------------
             %% 3. Check availability images          
             for iSubject = 1:x.dataset.nSubjects
-                SubjSess = (iSubject-1)*length(x.SESSIONS) + iSession;
-                xASL_TrackProgress(SubjSess, x.dataset.nSubjects*length(x.SESSIONS));
+                SubjSess = (iSubject-1)*length(listSessions) + iSession;
```

Comment:
I like the replacement of x.SESSIONS by listSessions

Reply:
ok ;)